docs/nodes/hardfork-process.md

```diff
@@ -0,0 +1,54 @@
+# Celestia Hardfork Process
+
+Blockchain networks often times need to upgrade with new features
+which require coordination work among the validators prior to activating
+the upgrades.
+
+This process is called a Hardfork or a network upgrade. In those events,
+the Celestia Labs team will be coordinating with the validators on the
+what they need to do in order to be ready for an upcoming hardfork.
+
+Hardforks are not backward-compatible with older versions of the network
+software which is why it is important that validators upgrade their software
+to continue validating on the network after the network upgrades.
+
+## General Process
+
+The general process can be broken down into several components:
+
+- Hardfork Specifications and Features (defined by description of features
+  and code implementation of those features).
+- Binary used to add those features (a new binary release with those features
+  will be provided by Celestia Node team in order for validators to upgrade
```

Comment:
I don't think Node team is accurate here. In other words the app/core team will likely release a celestia-app  binary that validators need to upgrade to.

```suggestion
  will be provided by Celestia team in order for validators to upgrade
```